lib/provider.js

```diff
@@ -174,6 +174,7 @@ const init = (providerConfig) => {
           console.info('File already exists. Try to remove it.');
           await this.delete(file);
         }
+        const asciiFileName = file.name.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
```

Comment:
Since this conversion will be lossy regardless of the normalization, I think it would be better to use the compatibility denormalization to support more use cases.
This is not recommended for arbitrary text, but this fits the use case described on the [Unicode Normalization Forms](https://unicode.org/reports/tr15/#Compatibility_Composite_Figure):

> They can be applied more freely to domains with restricted character sets.

In this case, USASCII

```suggestion
        const asciiFileName = file.name.normalize('NFKD').replace(/[\u0300-\u036f]/g, '');
```

Reply:
I didn't noticed the K missing. Thanks!